docs/spec/lightclient/detection/detection.md

```diff
@@ -169,7 +357,7 @@ false* then we have a *fork on the chain*.
 > fork. 
 
 
-#### **[TMBC-LC-FORK]**
+#### **[TMBC-LC-FORK.1]**
 Let *a*, *b*, *c*, be light blocks and *t* a time. We define
 *light-client-fork(a,b,c,t)* iff
    - *sign-skip-match(a,b,c,t) = false* and
```

Comment:
I think the condition should be slightly different: sign-skip-match(a,b,c,t) = true, sequ-rooted(b), b is unique and b != c.